paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentMethodsUI.kt

```diff
@@ -203,3 +216,37 @@ internal fun PaymentMethodUI(
         }
     }
 }
+
+@Composable
+private fun PaymentMethodIconUi(
+    iconRes: Int,
+    iconUrl: String?,
+    imageLoader: StripeImageLoader,
+    tintOnSelected: Boolean,
+    color: Color,
+) {
+    val colorFilter = if (tintOnSelected) ColorFilter.tint(color) else null
+
+    if (iconUrl != null) {
+        StripeImage(
+            url = iconUrl,
+            imageLoader = imageLoader,
+            contentDescription = null,
+            contentScale = ContentScale.Fit,
+            modifier = Modifier.padding(
+                top = Spacing.cardLeadingInnerPadding / 2,
+                start = Spacing.cardLeadingInnerPadding / 2,
```

Comment:
What’s the reason for using half the padding here?

Reply:
That's what makes it look right 🙈

Reply:
It’s weird that it is different from what we use for local assets. Both local assets and remote assets should be positioned in the same spot.